extensions/rich_text_components/Link/Link.js

```diff
@@ -25,38 +25,52 @@ oppia.directive('oppiaNoninteractiveLink', [
-          // This is done for backward-compatibility.
-          $scope.target = '_blank';
-        } else {
-          $scope.target = (
-            oppiaHtmlEscaper.escapedJsonToObj(
-              $attrs.openLinkInSameWindowWithValue) ? '_top' : '_blank');
-        }
+      controller: [
+        '$scope', '$attrs', 'explorationContextService', 'PAGE_CONTEXT',
+        'EDITOR_TAB_CONTEXT',
+        function(
+            $scope, $attrs, explorationContextService,
+            PAGE_CONTEXT, EDITOR_TAB_CONTEXT) {
+          if (!$attrs.openLinkInSameWindowWithValue) {
+            // This is done for backward-compatibility.
+            $scope.target = '_blank';
+          } else {
+            $scope.target = (
+              oppiaHtmlEscaper.escapedJsonToObj(
+                $attrs.openLinkInSameWindowWithValue) ? '_top' : '_blank');
+          }
 
-        var untrustedUrl = oppiaHtmlEscaper.escapedJsonToObj(
-          $attrs.urlWithValue);
-        if (untrustedUrl.indexOf('http://') !== 0 &&
-            untrustedUrl.indexOf('https://') !== 0) {
-          return;
-        }
-        $scope.url = untrustedUrl;
+          var untrustedUrl = oppiaHtmlEscaper.escapedJsonToObj(
+            $attrs.urlWithValue);
+          if (untrustedUrl.indexOf('http://') !== 0 &&
+              untrustedUrl.indexOf('https://') !== 0) {
+            return;
+          }
+          $scope.url = untrustedUrl;
 
-        $scope.showUrlInTooltip = false;
-        $scope.text = $scope.url;
-        if ($attrs.textWithValue) {
-          // This is done for backward-compatibility; some old explorations
-          // have content parts that don't include a 'text' attribute on
-          // their links.
-          $scope.text = oppiaHtmlEscaper.escapedJsonToObj($attrs.textWithValue);
-          // Note that this second 'if' condition is needed because a link may
-          // have an empty 'text' value.
-          if ($scope.text) {
-            $scope.showUrlInTooltip = true;
+          $scope.showUrlInTooltip = false;
+          $scope.text = $scope.url;
+          if ($attrs.textWithValue) {
+            // This is done for backward-compatibility; some old explorations
+            // have content parts that don't include a 'text' attribute on
+            // their links.
+            $scope.text =
+              oppiaHtmlEscaper.escapedJsonToObj($attrs.textWithValue);
+            // Note that this second 'if' condition is needed because a link may
+            // have an empty 'text' value.
+            if ($scope.text) {
+              $scope.showUrlInTooltip = true;
+            }
+            // This code prevents tabbing while in editor mode.
+          }
+          if (
+            explorationContextService.getPageContext() ===
+              PAGE_CONTEXT.EDITOR &&
+            explorationContextService.getEditorTabContext() ===
+              EDITOR_TAB_CONTEXT.EDITOR) {
+            $scope.tabIndexVal = '-1';
```

Comment:
Ah -- I'd suggest adding a linebreak after line 65, then, and putting the comment on the line after that break, just before the 'if' statement.